apple/RNCWebView.m

```diff
@@ -1143,9 +1143,13 @@ - (void)                  webView:(WKWebView *)webView
   
   if (_onShouldStartLoadWithRequest) {
     NSMutableDictionary<NSString *, id> *event = [self baseEvent];
+    if (request && request.mainDocumentURL) {
```

Comment:
There's other logic that would already blow up if `request` itself were `nil`, so I think you could just make this `if (request.mainDocumentURL) {`?

Reply:
Done